### PR TITLE
Container gump locations remembered and NetworkStats is now a gump

### PR DESCRIPTION
**Pull request by dissident76**
Container gumps saved in gumps.xml (bags, pouches, crates, etc) will now get restored correctly on logon.

### CODE REVIEW

File: src/Game/UI/Gumps/NetworkStatsGump.cs [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+        {
+            _trans.Width = Width = _label.Width + 20;
+            _trans.Height = Height = _label.Height + 20;
+
+            base.Update(totalMS, frameMS);
+        }
+
+        public override bool Draw(UltimaBatcher2D batcher, int x, int y)
+        {
+            GameScene scene = Client.Game.GetScene<GameScene>();
```
Comment (andreakarasho) [inline src/Game/UI/Gumps/NetworkStatsGump.cs:101]: Unused ?

File: src/Network/NetStatistics.cs [hunk 2]
```diff
@@ -47,6 +47,7 @@ internal class NetStatistics
 
         public uint Ping { get; private set; }
 
+        public RenderedText RenderedText { get; private set; }
```
Comment (andreakarasho) [inline src/Network/NetStatistics.cs:50]: 2 instances of RenderedText.
Just put
```csharp
public RenderedText RenderedText => _renderedText;
```

File: src/Game/UI/Gumps/NetworkStatsGump.cs (hunk 1 above)
Comment (dissident76) [reply src/Game/UI/Gumps/NetworkStatsGump.cs:101]: yeah, I did a cope and paste from DebugBump and forgot to delete some stuff

File: src/Network/NetStatistics.cs (hunk 2 above)
Comment (dissident76) [reply src/Network/NetStatistics.cs:50]: done

**Pull request closed without merging**